CMakeLists.txt

```diff
@@ -71,14 +71,14 @@ ENDIF()
 ### Search dependencies
 
 IF(USEQT5)
-  FIND_PACKAGE(Qt5Core REQUIRED)
-  FIND_PACKAGE(Qt5Widgets REQUIRED)
-  FIND_PACKAGE(Qt5Network REQUIRED)
-  FIND_PACKAGE(Qt5Concurrent REQUIRED)
-  FIND_PACKAGE(Qt5Help REQUIRED)
+  FIND_PACKAGE(Qt5Core 5.14 REQUIRED)
```

Comment:
This would break compatibility with Ubuntu 20.04, which ships Qt 5.12.8. We need a solution that is both forward- and backward-compatible.

Reply:
Fixed. I have set the minimum required Qt version to 5.12 and have dropped support for Qt 4.